app/models/training_tip.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class TrainingTip < ActiveRecord::Base
```

Comment:
I'm not seeing a relationship back to a user from this model.

Reply:
ok, I will add this relationship there

Reply:
done